src/vm/gcenv.ee.cpp

```diff
@@ -917,6 +924,14 @@ void GCToEEInterface::StompWriteBarrier(WriteBarrierParameters* args)
         g_card_bundle_table = args->card_bundle_table;
 #endif
 
+#if defined(_ARM64_)
+        g_lowest_address = args->lowest_address;
+        g_highest_address = args->highest_address;
```

Comment:
You still need the FlushProcessWriteBuffers here to handle the cases where these are accessed from C/C++.